tests/Shared/LayoutRenderers/AspNetRequestItemLayoutRendererTests.cs

```diff
@@ -0,0 +1,85 @@
+﻿using System.IO;
+#if !ASP_NET_CORE
+using System.Collections.Specialized;
+using System.Web;
+using System.Web.Routing;
+using System.Web.SessionState;
+#else
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Http.Features;
+using Microsoft.Extensions.Primitives;
+using HttpContextBase = Microsoft.AspNetCore.Http.HttpContext;
+#endif
+using NLog.Common;
+using NLog.Web.LayoutRenderers;
+using NSubstitute;
+using NSubstitute.ExceptionExtensions;
+using Xunit;
+using System.Collections.Generic;
+
+
+namespace NLog.Web.Tests.LayoutRenderers
+{
+    public class AspNetRequestItemLayoutRendererTests : TestInvolvingAspNetHttpContext
```

Comment:
Can this be renamed to `AspNetHttpContextItemLayoutRendererTests`

Reply:
Complete